include/GL/glui.h

```diff
@@ -34,11 +34,23 @@
 #ifndef GLUI_GLUI_H
 #define GLUI_GLUI_H
 
-// Having stdlib here first fixes some 'exit() redefined' errors on MSVC.NET
-// that come from old GLUT headers.
-#include <cstdlib>
+//Note: GCC is not required for this feature. It's just an indicator. 
+#ifdef __GNUC__
+//GLUI_GLX uses glXGetCurrentDisplay to find GLUT's chosen X display.
+//It will #include GL/glx.h and X11/Xlib.h. This covers all desktops.
```

Comment:
In combination with https://github.com/libglui/glui/pull/73 there needs to be a check here for Mac OSX.  See also: https://github.com/nigels-com/glui/commit/e3e43a4e91331504fdb3338a10f4260a4bec29a8